data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Much of the nature of the "assistance from anarchist planets" is redacted, but it seems it mostly involves shutting down groups that try to reveal or harm the Hai. However, there are notes about deals with anarchists who oppose slavery, arranging massive rescues, bringing thousands of slaves at a time to freedom in Hai space. Also, there is an amnesty program for pirates who have committed no serious crimes, which explains the abundance of reformed pirates on Allhome.`
+			`	Recent anti-pirate actions by the Free Worlds and Republic Navy have weakened the large anarchist organizations that had deals with the Hai. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations, haven't worked with small pirate groups.`
+			`	They can't track small pirate groups, but they clearly have an intelligence network. Is it good for anything?`
+				goto reading
+				to display
+					not "Hai Reveal [A11] Sound the Alarm: human book"
+			`	It seems that, although the Hai have no trouble with large, established, hierarchies, they have trouble tracking small, independent, anarchist, groups. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: human book"
+
+			label unfettered
+			action
+				log `A Hai intelligence report puts the cost to Hai of the Unfettered raids at 3.5 billion credits a day, while the cost is expected to double in "full Hai unification." Unification costs include 5 billion credits a day of "military aid."`
+				log `A Hai intelligence report on "Unfettered military activities" included several fully-redacted sections, including "Multi-Decadal Timeline Expected before Unfettered Progress," and "Theories on the Sentient Evolution of Nocturnal Raptors."`
+				set "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report begins with an economic analysis of the cost of raids at Wah Ki, which it places at a staggering 3.5 billion credits per day. This is 1.5 billion impact on trade; 1.5 billion for military fleet repairs, replacements, ammunition, and infrastructure to support it; and an estimated 500 million in humanitarian aid to the Unfettered, to prevent the problem from worsening. It compares that to "full Hai unification" for which the cost is 7 billion credits per day: 3 billion in humanitarian aid to the Unfettered, 5 billion in military aid, but 1 billion per day recouped through improved trade.`
```

Comment:
How does unification cost more? What is this unification?
Hai+unfettered if unfettered invade?

Reply:
Unification means no barriers to funding the Unfettered. The Unfettered military is the Hai military again (or, at least, the most competent part of it) and they'll properly funded as such. The Fettered have the responsibility for paying all social welfare costs of the Unfettered. That's expensive. However, removing the barrier to trade at Wah Ki will make trade more efficient. This is roughly patter ended off of Korean Unification, which has similarly staggering costs (albeit not military).

Reply:
Idea is ok it prob could be clarified though

Reply:
It could be clarified, but it shouldn't be. That report is supposed to be vague and ominous.